systemd/Dockerfile.fedora.tpl

```diff
@@ -0,0 +1,47 @@
+FROM #{FROM}
+
+RUN rpm --rebuilddb && yum install -y \
+        ca-certificates tar \
+        systemd \
+    && yum clean all
+
+# Install Systemd
+
+ENV container docker
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
```

Comment:
I did `systemd disable` lot of services but still got tons of noisy logs when systemd start then I found this article about systemd on fedora container http://developerblog.redhat.com/2014/05/05/running-systemd-within-docker-container/. Basically, this is a fast and clean way to stop systemd from stating many services (we only keep  systemd, journald, udev and resin services running), it's way better than disable every service since this image is not optimised for Docker container.
Until they release the armhf fedora Docker images, I think we should follow this way